tests/portchannel_test.py

```diff
@@ -127,6 +127,32 @@ def test_add_portchannel_with_invalid_fast_rate(self, fast_rate):
         assert result.exit_code != 0
         assert 'Invalid value for "--fast-rate"'  in result.output
 
+    @pytest.mark.parametrize("static", ["False", "True", "false", "true"])
+    def test_add_portchannel_with_static_adhoc_validation(self, static):
+        config.ADHOC_VALIDATION = True
+        runner = CliRunner()
+        db = Db()
+        obj = {'db':db.cfgdb}
+
+        # add a portchannel with fats rate
+        result = runner.invoke(config.config.commands["portchannel"].commands["add"], ["PortChannel0101", "--static", static], obj=obj)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
```

Comment:
The test should not just check the return code but also validate that when --static is set, config_db is set with fv static, true. Can you please update the test to reflect this?